.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+## Bug Description
```

Comment:
Have you thought about adding in a "expected behavior" section? This helps to quickly identify if the reporter is reporting an actual bug, or if it is a misinterpretation of the feature.